cmd/producer/main.go

```diff
@@ -1,22 +1,24 @@
 package main
 
 import (
+	"time"
+
 	"github.com/ONSdigital/dp-observation-importer/event"
 	"github.com/ONSdigital/dp-observation-importer/schema"
 	"github.com/ONSdigital/go-ns/kafka"
-	"time"
 )
 
 func main() {
 	var brokers []string
 	brokers = append(brokers, "localhost:9092")
-
+	// instanceID := "a4695fee-f0a2-49c4-b136-e3ca8dd40476"
 	producer := kafka.NewProducer(brokers, "observation-extracted", int(2000000))
-
-	event1 := event.ObservationExtracted{InstanceID: "7", Row: "5,,sex,male,age,30"}
-	sendEvent(producer, event1)
-	event2 := event.ObservationExtracted{InstanceID: "7", Row: "5,,sex,female,age,20"}
+	// event1 := event.ObservationExtracted{InstanceID: "7", Row: "5,,sex,male,age,30"}
+	// sendEvent(producer, event1)
+	event2 := event.ObservationExtracted{InstanceID: "a4695fee-f0a2-49c4-b136-e3ca8dd40476", Row: ",,,,,,,"}
```

Comment:
This ID is hardcoded - presumably this should be coming from somewhere?

Reply:
This is the testing doc